src/main/sensors/battery.c

```diff
@@ -270,42 +269,34 @@ void updateCurrentMeter(int32_t lastUpdateAt, rxConfig_t *rxConfig, uint16_t dea
 
             updateCurrentDrawn(lastUpdateAt);
 
-            updateConsumptionWarning();
-
             break;
         case CURRENT_SENSOR_VIRTUAL:
-            amperageLatest = (int32_t)batteryConfig->currentMeterOffset;
+            amperage = (int32_t)batteryConfig->currentMeterOffset;
```

Comment:
Why did you change this? This has got nothing to do with ESC feedback. It also breaks consumption tracking.